iroh-bitswap/src/lib.rs

```diff
@@ -512,7 +512,7 @@ impl<S: Store> NetworkBehaviour for Bitswap<S> {
     ) -> Poll<NetworkBehaviourAction<Self::OutEvent, Self::ConnectionHandler>> {
         inc!(BitswapMetrics::NetworkBehaviourActionPollTick);
         // limit work
-        for _ in 0..100 {
+        for _ in 0..50 {
```

Comment:
let's extract these real quick as const and I can play with magic numbers eventually.

Reply:
can I do this in another PR? I would like CI to finish today..

Reply:
Tbh not that important, we got lot's of magic numbers I want to eventually extract and then play with configs and do regression testing against benchmarks... eventually.